File: dataset/human/c/file_9295.c
#include <stdio.h>

int main(void){

  int N, K;
  scanf("%d %d",&N,&K);
  int h[N], i;
  for(i=0;i<N;i++){
    scanf("%d",&h[i]);
  }
  int j, count=0;
  for(j=0;j<N;j++){
    if(h[j]>=K){
      count+=1;
    }
  }
  printf("%d",count);
  return 0;
}

File: dataset/human/java/file_9295.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = Integer.parseInt(sc.next());
        int count = 0;
        String ans = "No";
        for (int i = 0; i < n; i++) {
            count = sc.next().equals(sc.next()) ? count + 1 : 0;
            if(count == 3) {
                ans = "Yes";
            }
        }

        System.out.println(ans);
    }
}
